pkg/polaris/ctl_polaris.go

```diff
@@ -124,13 +126,12 @@ func (ctl *CRSpecBuilderFromCobraFlags) AddCRSpecFlagsToCommand(cmd *cobra.Comma
 	cmd.Flags().StringVar(&ctl.Registry, "registry", ctl.Registry, "Docker registry e.g. docker.io/myuser")
 
 	// Those flags need be hidden since external databases are not currently supported
-	cmd.Flags().StringVar(&ctl.PostgresHost, "postgres-host", ctl.PostgresHost, "")
-	cmd.Flags().IntVar(&ctl.PostgresPort, "postgres-port", ctl.PostgresPort, "")
-	cmd.Flags().MarkHidden("postgres-host")
-	cmd.Flags().MarkHidden("postgres-port")
-
+	cmd.Flags().StringVar(&ctl.PostgresHost, "postgres-host", GetPolarisDefault().PolarisDBSpec.PostgresDetails.Host, "Postgres Host")
+	cmd.Flags().IntVar(&ctl.PostgresPort, "postgres-port", GetPolarisDefault().PolarisDBSpec.PostgresDetails.Port, "PostgresPort")
 	cmd.Flags().StringVar(&ctl.PostgresUsername, "postgres-username", ctl.PostgresUsername, "Postgres username")
 	cmd.Flags().StringVar(&ctl.PostgresPassword, "postgres-password", ctl.PostgresPassword, "Postgres password")
+	cmd.Flags().StringVar(&ctl.PostgresSSLMode, "postgres-ssl-mode", string(GetPolarisDefault().PolarisDBSpec.PostgresDetails.SSLMode), "Postgres ssl mode [disable, allow, prefer, require]")
+	cmd.Flags().BoolVar(&ctl.PostgresInternal, "postgres-container", GetPolarisDefault().PolarisDBSpec.PostgresDetails.IsInternal, "Not recommended. Use a PostgreSQL container instead of an external PostgreSQL instance.")
```

Comment:
Are we recommending to use postgreSQL container instead of external database?

Reply:
We recommend external databases but the message was indeed very confusing. It has been updated